C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+Description : In a given string, the task is to create a string
+              with the first letter of every word in the string.
```

Comment:
Quite vague. Can you be more descriptive?

Reply:
It's a question present in gfg , and they also provided the same description ,
You can check the link - https://practice.geeksforgeeks.org/problems/print-first-letter-of-every-word-in-the-string3632/1

Reply:
Still I will try to explain it with example